webapp/src/components/user_popover/badge_list.scss

```diff
@@ -0,0 +1,24 @@
+#badgeList {
```

Comment:
We'll want to put this in a container to make sure it doesn't conflict with a `badgeList` id in the future. It's unlikely but it's standard for plugins to do something like this:

```scss
.badgesPlugin {
    .badgeList {
        ...
    }
}
```

Reply:
Good point, I included `badgesPlugin` id to prevent potential conflicts in the future